structuredefinitions/Extension-England-RemovalReason.xml

```diff
@@ -0,0 +1,61 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-England-RemovalReason" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-England-RemovalReason" />
+  <version value="1.0.0" />
+  <name value="ExtensionEnglandRemovalReason" />
+  <title value="Extension England Removal Reason" />
```

Comment:
Need to be: Extension England FGM Removal Reason

Reply:
Agree. Please modify